### DISCUSSION TITLE
Rename/nickname devices

### DISCUSSION THREAD

**Discussion by kevinawoo** [AppGround-io/bluetooth-keyboard-and-mouse-support#128] [Ideas]
I have a bunch of iPads that connect to my phone as `iPad`. It's hard to tell which is which. 

Some meta data would help, like last paired time or last connected time. 

The ability to give a device a name in the app would be really awesome too. 

Here's a screenshot shot of what I mean:
![Screenshot_20190816-000609](https://user-images.githubusercontent.com/3469532/63149685-afc09680-bfb9-11e9-926a-ec59eea00822.png)

I have no idea which "iPad" goes to which.

Also, would you be able to condense the view a tad bit? When there's a lot of devices connected, I kind of loose my place when scrolling, especially when every other one is named `iPad`.

**Comment by manuelkue** [comment]
I'd like to add another feature that should fit to this one:
It would be great to be able to hide known bluetooth devices that have no relation to this app as currently all known devices are listed.
Thanks in advance! :-)

![Screenshot_20201122-165044_Bluetooth Keyboard   Mouse](https://user-images.githubusercontent.com/44258924/99908426-e324c180-2ce2-11eb-8f6b-4d097ad89fe8.jpg)
